amp/repository/aim/view/teamAuditList.jsp

```diff
@@ -215,6 +221,11 @@ function showUser(email){
 											</c:if>
 										</b>
 									</td>
+									<td background="/TEMPLATE/ampTemplate/img_2/ins_bg.gif" class="inside">
+                                    <b class="ins_title">
+                                    <digi:trn key="aim:action"> View differences</digi:trn>

```

Comment:
you have a space, in View differences please remove it to avoid it to be treated as another string (i dont recall if we are trimming )